suite-common/transaction-cache-engine/src/types.ts

```diff
@@ -0,0 +1,37 @@
+import BigNumber from 'bignumber.js';
+
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { AccountTransaction } from '@trezor/connect';
+
+export type AccountUniqueKey = string & { __accountKey: 'AccountKey' };
```

Comment:
Could be changed to template literal type:
```suggestion
export type AccountUniqueKey = `${NetworkSymbol}-${Account["descriptor"]}` & { __accountKey: 'AccountKey' };
```

for better string type integrity.

Reply:
that is cool